.github/workflows/remainder-bot.yml

```diff
@@ -0,0 +1,77 @@
+#####################################################################################
+# Remainder Bot
+#
+# Workflow starts when:
+# 1) every day at 00:00 UTC
+#
+# Workflow:
+# 1) Check and close inactive issues and PRs
+# 2) Remind user to provide the requested feedback for issues
+# 3) Remind user to provide the requested feedback for PRs
+# 4) If no response is received in the next 15 days, we will close this issue
+# 5) If no response is received in the next 15 days, we will close this pull request
+# 6) Close the issue with a message after 60 days
+# 7) Close the pull request with a message after 60 days
+#####################################################################################
+
+name: "feedback-reminder-bot"
+
+on:
+  schedule:
+    # every day at 00:00 UTC
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+concurrency:
+  group: feedback-reminder-bot
+  cancel-in-progress: true
+jobs:
+  close-inactive-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check and close inactive issues and PRs
+        uses: actions/stale@v9
+        with:
+          any-of-labels: "awaiting response,awaiting changes"
+
+          days-before-stale: 15
+          stale-issue-label: "response requested"
```

Comment:
>Target issues labelled with awaiting-response

`stale-issue-label: "response requested"` means that we will add a label called `response requested` to all stale issues. This is not consistent with the PR description and what has been agreed upon in the pull request.

I would expect us to have `only-issue-labels: awaiting-response` based on the PR description. Please read through the docs for this action, and update all input arguments to match our criteria: https://github.com/actions/stale?tab=readme-ov-file#list-of-input-options

Reply:
Done